kf_lib_data_ingest/etl/accounting/test.py

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
This module defines all study wide accounting tests. These can be imported in the accounting configs